make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -207,3 +207,20 @@ UPDATE quota_usage SET used = used - 'count';
 /* make Clair and Trivy as reserved name for scanners in-tree */
 UPDATE scanner_registration SET name = concat_ws('-', name, uuid) WHERE name IN ('Clair', 'Trivy') AND immutable = FALSE;
 UPDATE scanner_registration SET name = split_part(name, '-', 1) WHERE immutable = TRUE;
+
+/*update event types in table 'notification_policy'*/
+UPDATE notification_policy SET event_types = '["DOWNLOAD_CHART","DELETE_CHART","UPLOAD_CHART","DELETE_ARTIFACT","PULL_ARTIFACT","PUSH_ARTIFACT","SCANNING_FAILED","SCANNING_COMPLETED"]';
```

Comment:
This will update all notification policies in the system to these eight event types, even the policy only has one event type.

Reply:
In the previous version, customers can not select the event types for the webhook policy, so just set it to all supported event types is ok.